index.js

```diff
@@ -32,9 +33,7 @@ module.exports = {
   },
 
   shouldIncludeFiles: function() {
-    var config = this.app.project.config()['ember-cli-mirage'];
-
-    return config.force || (this.app.env !== 'production')
+    return this.addonConfig.force || !this.addonConfig.usingProxy ? this.addonConfig.isEnabledEnvironment : true;
```

Comment:
ah gotcha. Yah the reason I originally changed it here was bc I thought I needed to include the files into the build, since even tho `--proxy` may be passed the user would still want it for testing. But if there's a separate build with `env == test` during `ember s`, this should work